ixdiagnose/artifacts/logs.py

```diff
@@ -0,0 +1,20 @@
+from .base import Artifact
+from .items import Directory, File, Pattern
+
+
+class Logs(Artifact):
+    base_dir = '/var/log'
+    name = 'logs'
+    individual_item_max_size_limit = 10 * 1024 * 1024
+    items = [
+        Directory('openvpn'),
+        Directory('pods'),
+        Directory('samba4'),
+        Pattern('daemon.+'),
+        File('failover.+'),
```

Comment:
Did you want to use `Pattern` here?

Is there any reason we are changing behavior from old ixdiagnose to whitelisting files? I miss `auth.log` here for example

Reply:
It was explicitly requested that we specify which log files we want to retrieve..
Adding `auth.log` to the list